source/image-handler/test/image-request/parse-image-bucket.spec.ts

```diff
@@ -126,4 +126,30 @@ describe("parseImageBucket", () => {
       });
     }
   });
+
+  it("should parse bucket-name from first part in thubor request but fail since it's not allowed", () => {
```

Comment:
`thumbor` i spelled incorrectly in the above string and in the next test.

Does this test cover when no bucket is provided? For instance, in my use case I do not want to specify a bucket name in the URL.

Reply:
Ok thanks for the reply.

Yes, I agree with using a keyword like "bucket:"/ "s3:" similar to the "filters:" in the URL. That would work around the issue and hopefully not cause an issue with the path/bucket name

Example
```
    const event = { path: "/filters:grayscale()/bucket:test-bucket/test-image-001.jpg" };
    const event = { path: "/filters:grayscale()/s3:test-bucket/test-image-001.jpg" };
```

Reply:
I kinda don't like how it's not part of the `ThumborMapper` class, but it's kinda not part of thumbor itself and also needs to happen at an earlier stage, so it makes sense to have it where it is now..

Reply:
Hey @t4colingough any update?

Reply:
Hi @bennet-esyoil, 

This change has been made internally and is currently set to be included in the next minor/major release. While our internal processes had us merge the code manually, once the release with this change is out, you'll see yourself included in the external contributors section towards the bottom of the readme and this PR will be closed.

Thanks for your contributions,
Simon